src/gql/queries/get-commit-queue.ts

```diff
@@ -0,0 +1,42 @@
+import gql from "graphql-tag";
+import { Patch } from "../../types/patch";
+export const GET_COMMIT_QUEUE = gql`
+  query CommitQueue($id: String!) {
+    commitQueue(id: $id) {
+      ProjectID
+      Queue {
+        Issue
+        EnqueueTime
+        Patch {
```

Comment:
must have not caught these in your backend PR - field names should never be uppercased

Reply:
just opened up a pr for this 
https://github.com/evergreen-ci/evergreen/pull/3485